hack/gosec.sh

```diff
@@ -0,0 +1,6 @@
+source hack/common.sh
+
+
+echo "Run go sec in pkg"
+cd $KUBEVIRT_DIR/pkg
+gosec -exclude-dir=testutils -fmt=junit-xml -out=output.xml ./... 
```

Comment:
First of all you should set output to something like this:

` -out=${ARTIFACTS-_out/artifacts}/output.xml`

Also we might consider using 
```
  -severity string                        
        Filter out the issues with a lower severity than the given value. Valid options are: low, medium, high (default "low")
```

Maybe we can introduce an env var (needs to be whitelisted in Makefile also) such as `${KUBEVIRT_GOSEC_SEVERITY-low}`, then we would be able to adjust it in the job without changing the script in kubevirt.